docs/platform/concepts/service-power-cycle.md

```diff
@@ -1,83 +1,153 @@
--   The service information and configuration will be stored on Aiven
-    Platform, while service data will be lost if there's no backup
-    available.
--   If the service has **time-based** or **PITR (point in time
-    recovery)** backups, they will be kept on Aiven Platform. The
-    backups are listed on the **Backups** page of the service in [Aiven
-    Console](https://console.aiven.io). Absence of the **Backups** view
-    means the service has no backups. For details on backups for
-    different Aiven services on different plans, refer to
-    [Backups at Aiven](/docs/platform/concepts/service_backups).
-
-:::warning
-Aiven does periodic cleanup of powered-off services on services powered
-off for longer than **180** consecutive days. Notification emails will
-be sent before actions are taken.
+[Aiven CLI](/docs/platform/concepts/service-power-cycle).
+
+## Power off a service
+
+:::warning[Kafka services]
+- For Kafka services with backups: Topic configuration, schemas and connectors are all
+  backed up, but not the data in topics. All topic data is lost on power off.
+
+- For Kafka services without backups: Topic configurations including all
+  topic data is lost on power off.
 :::
 
--   The message in the **Confirm power off** window will give some hints
-    on the consequence of the power off. For example, powering off an
-    Aiven service can erase data since the latest backup because the
-    service only has time-based but not PITR backups.
--   On the **Backups** page, hovering over the help
-    icon displays some details on the content of
-    the backups including what can be restored if the
-    service is powered on later.
-
-:::warning
-For backup enabled Aiven for Apache Kafka® services, topic
-configuration, schemas and connectors are all backed up, but not the
-data in topics. Therefore all topic data will be lost on power off. For
-Kafka services without backups, topic configurations including all
-topic data will be lost on power off.
+<Tabs groupId="sync">
+<TabItem value="Console" label="Console" default>
+
+1. From your project, on the left-side menu, click <Services className="icon"/> **Services**.
+1. For each the service to power, click <ActionsIcon className="icon"/> **Actions** >
+   **Power off service**
+
+</TabItem>
+<TabItem value="CLI" label="CLI">
+
+1. Run:
+
+   ```bash
+   avn service update <your service name> --power-off
+   ```
+
+</TabItem>
+</Tabs>
+
+Whenever a service is powered off:
+
+- All virtual machines of the service are **removed** from the
+  public cloud.
+- The service information and configuration are stored on Aiven
+  Platform. All service data is lost if there's no backup available.
+- If the service has **time-based** or **PITR (point in time
+  recovery)** backups, the backups are kept on the Aiven Platform.
+
+  See [Backups at Aiven][backup].
+
+:::important
+<AutoDelete/>
 :::
 
-## Power on
+<StaticIp/>
+
+When a service is powered off, you can [delete it]
```

Comment:
Why the brackets? Also the full stop is missing.

Reply:
I forgot the link 🙃 thanks